src/lines/LineMaterial.ts

```diff
@@ -304,7 +304,7 @@ class LineMaterial extends ShaderMaterial {
         get: function (): number {
           return this.uniforms.linewidth.value
         },
-        set: function (value: number) {
```

Comment:
I don't know why this wasn't already an error but whatever, fixed it. Also someone should (later, not in this PR) move this over to the class definition